dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -4,11 +4,18 @@ class Pd::ProfessionalLearningLandingController < ApplicationController
   def index
     # Get the courses that this user teaches
     workshops = Pd::Workshop.enrolled_in_by(current_user)
+    ended_workshops = workshops.in_state(Pd::Workshop::STATE_ENDED)
+
     courses_teaching = workshops.pluck(:course).uniq
```

Comment:
outside this PR, but this variable name is misleading. These are courses for workshops this teacher is enrolled in, but not necessarily teaching. It can include workshops that haven't occurred yet, and completed workshops that the teacher has not (yet) taught.